### PR TITLE
Changed colors to improve visibility in dark theme

### PR DESCRIPTION
**Pull request by adityaruplaha**
This color appears nicer with better visibility.

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3014)

### CODE REVIEW

**Comment by j-selby** [comment]
UI screenshot?

**Comment by Simonx22** [comment]
Windows screenshot:
image

**Comment by adityaruplaha** [comment]
Dont merge this right now please, i wanna make some more changes

**Comment by adityaruplaha** [comment]
Yup, blood red, black & yellow.

File: dist/qt_themes/qdarkstyle/style.qss [hunk 1]
```diff
@@ -740,6 +740,12 @@ QLabel
     border: 0px solid black;
 }
 
+QLabel::web_token_info_link,QLabel::web_signup_link
+{
+    color: #FFD700;
```
Comment (adityaruplaha) [inline dist/qt_themes/qdarkstyle/style.qss]: Doesn't work properly now. Ill fix it.

**Comment by adityaruplaha** [comment]
The link won't work now, need to edit the inline.

**Comment by BreadFish64** [comment]
so if I want to change the link colors in dark mode for #3005 I will have to change the the theme file for those specific links, correct?

**Comment by adityaruplaha** [comment]
Exactly, but how ?

**Comment by adityaruplaha** [comment]
I would like to have it in inline styling. I can also consider creating a new `QLink` class. Which one should I do ?

**Comment by adityaruplaha** [comment]
I am using inline styling to change link color.

File: src/citra_qt/configuration/configure_web.cpp [hunk 2]
```diff
@@ -21,18 +21,37 @@ ConfigureWeb::ConfigureWeb(QWidget* parent)
 
 ConfigureWeb::~ConfigureWeb() {}
 
+std::string getColor()
```
Comment (B3n30) [inline src/citra_qt/configuration/configure_web.cpp]: I don't like gow this is handeled. Adding new themes should not require to change the source code

Comment (jroweboy) [reply src/citra_qt/configuration/configure_web.cpp]: @B3n30 i agree, but you can't add the link styling to the stylesheet, and we are going to need a way to load different custom icons per stylesheet at some point anyway. So while i don't like the idea of having a code class to load some style sheet stuff, at some point we are going to need a more clever design to handle these two cases (links and custom icons)

**Comment by adityaruplaha** [comment]
I'm dropping the links this until we get a proper style handler.

**Comment by adityaruplaha** [comment]
please merge this for now.

**Comment by adityaruplaha** [comment]
Note, I will not add any further commits. So please review & merge if u want to asap.

**Comment by adityaruplaha** [comment]
Hey, are you merging this or not? Atleast say something...

**Comment by adityaruplaha** [comment]
#2999

**Comment by adityaruplaha** [comment]
Oh, sorry thats not it. Anyways, if it doesn't fit, i'll just close this.

**Pull request closed without merging**